docker-compose-rule-core/src/main/java/com/palantir/docker/compose/execution/DockerCommandLocations.java

```diff
@@ -1,47 +1,75 @@
 /*
  * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
```

Comment:
Are we keeping or removing the License? It's removed here but kept in the following file.